meta-mender-toradex-nxp/recipes-bsp/mender-tezi-metadata/mender-tezi-metadata_1.0.bb

```diff
@@ -0,0 +1,24 @@
+DESCRIPTION = "Toradex Easy Installer Metadata"
+
+inherit deploy
+
+SRC_URI = " \
+    file://LICENSE \
```

Comment:
The LICENSE text seems to be generic so should be enough to use:

```
LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
```

Then you can remove this file.